WEB-421 Notification on Like

selvex opened the pull request:
Currently when someone likes your video you will have to check on your own.
To make "liking" someones programms more engaging
we are now sending notifications to alert program owners if someone
likes their programs.

Commit:
WEB-421 Notification on Like

Currently when someone likes your video you will have to check on your own.
To make "liking" someones programms more engaging
we are now sending notifications to alert program owners if someone
likes their programs.

dmetzner commented:
*added*

- php7 rebase

- LikeNotification:
      - nomore hardcoded translations
      - simpler design -> using bs4 cards
- Notification system rework:
      - more than 1 notification possible
      - removing like removes notification if unseen
      - order === newest first
      - summary of total notifications added
      - summary + mark-all-as-read Button only visible when min 1 notification
      - profile avatar added for like/comment
      - responsive design fixes
      - improved design for notifiaction types
      - badges use now bs4 badges
      - badges more than 99 notification -> show 99+
      - JS rework, code redundancy removed, etc.


- added translations
- added more notification feature tests
- added notification badge tests
- footer fix: all corresponding css now in feature.scss

dmetzner committed:
- Translator Controller update: added transchoice
    - LikeNotification:
      - nomore hardcoded translations
      - simpler design -> using bs4 cards
    - Notification system rework:
      - more than 1 notification possible
      - removing like removes notification if unseen
      - order === newest first
      - summary of total notifications added
      - summary + mark-all-as-read Button only visible when min 1
        notification
      - profile avatar added for like/comment
      - responsive design fixes
      - improved design for notifiaction types
      - badges use now bs4 badges
      - JS rework, code redundancy removed, etc.

    - added translations
    - added more notification feature tests
    - added notification badge tests
    - footer fix: all corresponding css now in feature.scss
